internal/aez/general.go

```diff
@@ -73,6 +73,11 @@ const (
 	PM RID = 30
 )
 
+// IsCCD returns if the RID is for a CCD
+func (rid RID) IsCCD() bool {
+	return rid == CCD1 || rid == CCD2 || rid == CCD3 || rid == CCD4 || rid == CCD5 || rid == CCD6 || rid == CCD7
+}
+
 func (rid RID) String() string {
```

Comment:
Why not pointers in these two?

Reply:
well I guess I can make it of *everything* but they're just scalar values and probably not copied as many times.